Assets/Scripts/SpaceSuitManager.cs

```diff
@@ -67,7 +69,18 @@ private void Heal(int heal, HealthEffectSource source)
     
     private void Damage(int dmg, HealthEffectSource source)
     {
-        SetHealth(health - dmg, source);
+        if (killOnNextHit)
```

Comment:
Instead of Kill on Next hit, just make an alien firing squad health effect that does a ton of damage - add a new source to health effect source

Reply:
yeah chuck and i wrote this hacky shit before you added the source stuff

Reply:
Fix it